### DISCUSSION TITLE
[Feature] open shared link in mobile app / "shared with you"

### DISCUSSION THREAD

**Discussion by traktuner** [immich-app/immich#12660] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I frequently send and receive Immich share links. It would be fantastic if, when Immich is installed on a mobile device, these links automatically opened in the Immich app instead of in a web browser.

Additionally, it would be great to have a "Shared with You" section in the app where I can easily view the photos and videos shared with me. This should ideally work both for content on my own server and for files shared from other Immich servers.

Thank you!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2327
